Kickstarter-iOS/Views/PledgeScreenCTAContainerView.swift

```diff
@@ -0,0 +1,189 @@
+  }()
+
+  private lazy var spacer: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureSubviews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> \.layoutMargins .~ .init(all: Styles.grid(3))
+
+    _ = self.applePayButton
+      |> applePayButtonStyle
+
+    _ = self.ctaStackView
+      |> ctaStackViewStyle
+
+    _ = self.disclaimerLabel
+      |> disclaimerLabelStyle
+
+    _ = self.layer
+      |> checkoutLayerCardRoundedStyle
+      |> \.backgroundColor .~ UIColor.white.cgColor
+      |> \.shadowColor .~ UIColor.black.cgColor
+      |> \.shadowOpacity .~ 0.12
+      |> \.shadowOffset .~ CGSize(width: 0, height: -1.0)
+      |> \.shadowRadius .~ CGFloat(1.0)
+      |> \.maskedCorners .~ [
+        CACornerMask.layerMaxXMinYCorner,
+        CACornerMask.layerMinXMinYCorner
+      ]
+
+    _ = self.pledgeCTAButton
+      |> greenButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in Strings.Pledge() }
+
+    let isAccessibilityCategory = self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+
+    _ = self.rootStackView
+      |> adaptableStackViewStyle(isAccessibilityCategory)
+  }
+
+  // MARK: - View Model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+  }
+
+  // MARK: - Configuration
+
+  // TODO: Will be addressed in functionality PR
+//  func configureWith(value: ) {
+//  }
+
+  // MARK: Functions
+
+  private func configureSubviews() {
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([self.pledgeCTAButton, self.applePayButton], self.ctaStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.disclaimerLabel, self.disclaimerView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([self.ctaStackView, self.disclaimerView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.pledgeCTAButton.addTarget(
+      self, action: #selector(self.pledgeButtonTapped), for: .touchUpInside
+    )
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.pledgeCTAButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.pledgeCTAButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
+      self.applePayButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.applePayButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
+      self.disclaimerView.leadingAnchor.constraint(
```

Comment:
Instead of pinning the leading/trailing we could use layout margins on the stackview. the label shouldn't need a container view.